packages/tools/pluggable-widgets-tools/scripts/validate-copyright.js

```diff
@@ -0,0 +1,24 @@
+const { readFile } = require("fs").promises;
+const { join } = require("path");
+const { extractParts } = require("./copyright-utils");
+
+main().catch(e => {
+    console.error(e);
+    process.exit(1);
+});
+
+async function main() {
+    const packageJsonPath = join(process.cwd(), "package.json");
+    const packageJson = JSON.parse((await readFile(packageJsonPath)).toString());
+    const copyright = packageJson.copyright;
+
+    const [firstTextPart, endYear, secondTextPart] = extractParts(copyright);
+
+    if (
+        firstTextPart !== "Copyright © 2005-" ||
```

Comment:
Again, too specific for shipping with piw tools.

Reply:
This should be loosened up and a correct string set if the property isnt formed well or doesnt exist